testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -142,46 +138,45 @@ def test_old_SegmentGroup_init_works():
 
 class TestAtomGroupToTopology(object):
     """Test the conversion of AtomGroup to TopologyObjects"""
-    @dec.skipif(parser_not_found('DCD'),
-                'DCD parser not available. Are you using python 3?')
-    def setUp(self):
-        self.u = mda.Universe(PSF, DCD)
-
-    def tearDown(self):
-        del self.u
+    @pytest.mark.skipif(parser_not_found('DCD'),'DCD parser not available. Are you using python 3?')
```

Comment:
dcd is available on python 2 as a parser. Those tests should run for you. But since we soon have a python3 DCD parser I'm not to worried if the tests are skipped but they should be run on phython2.